### ISSUE TITLE
vue.runtime.common.js:2961 Uncaught ReferenceError: _h is not defined

### ISSUE THREAD

**Issue by nizulzaim** [Akryum/meteor-vue-component#92]
Hi, when updating to vue 2.1.6, i encounter this error. When i read this page, [Vue 2.1.0](https://github.com/vuejs/vue/releases/tag/v2.1.0), it might be problem cause by changing of filename. You can see in **Dist Files Adjustments** section. I try to lookup anything related to their suggestion but cannot find anything related. I hope you continue to working on this project although small fix for every week. I really love Meteor + Vue & i cannot use it without your package. Thanks.

Right now their suggestion is 

```
// before
resolve: {
  alias: {
    vue$: 'vue/dist/vue.js'
  }
}

// after
resolve: {
  alias: {
    vue$: 'vue/dist/vue.common.js'
  }
}
```

I dont know it is related to this change. Hope you can see on this problem. Thanks.

**Comment by Akryum** [comment]
Fixed in akryum:vue-component@0.7.8

**Comment by Akryum** [comment]
If someone run into an issue, try to update both the `vue` npm package and the `akryum:vue-component` meteor package to their latest version.
